### PR TITLE
Add test for selecting nodes on mac

### PR DESCRIPTION
**Pull request by christian-byrne**
Add tests from https://github.com/Comfy-Org/litegraph.js/pull/182 (drag-select on Mac)

### CODE REVIEW

File: browser_tests/ComfyPage.ts [hunk 1]
```diff
@@ -914,13 +924,15 @@ export class ComfyPage {
     return new NodeReference(id, this)
   }
   async getNodeRefsByType(type: string): Promise<NodeReference[]> {
```
Comment (christian-byrne) [inline browser_tests/ComfyPage.ts:926]: Assumed this is intended return type based on annotation

**Review by huchenlei — APPROVED** [review]

**Comment by huchenlei** [comment]
@christian-byrne Do you have any idea why the test fails even after https://github.com/Comfy-Org/litegraph.js/pull/182 has been merged?

**Comment by christian-byrne** [comment]
I thought it was because litegraph version wasn't updated yet

**Comment by huchenlei** [comment]
> I thought it was because litegraph version wasn't updated yet

The commit should be included in v0.7.81 litegraph which was updated yesterday in https://github.com/Comfy-Org/ComfyUI_frontend/pull/1050

**Comment by christian-byrne** [comment]
Not sure. I can fix when home 6 pst

**Comment by christian-byrne** [comment]
- **Ctrl+Drag Multiple Node Selection Test**: Updated the test to draw a selection box around both clip nodes instead of just one, as tnode query not stable
- **Ctrl+Click to Add Node to Selection Test**: I initially believed this feature was already supported on Mac, but it turns out it wasn't. I submitted a PR to add this feature and moved the test case there: https://github.com/Comfy-Org/litegraph.js/pull/189

**Pull request merged**